pywebdriver/plugins/mettler_toledo_8217.py

```diff
@@ -0,0 +1,109 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import logging
+import re
+import termios
+
+import serial
+
+from .scale_driver import AbstractScaleDriver
+
+ANSWER_RE = re.compile(rb"^\?(?P<status>.)|(?P<weight>\d+\.\d+)$")
+
+_logger = logging.getLogger(__name__)
+
+class MettlerToledo8217ScaleDriver(AbstractScaleDriver):
```

Comment:
please note that the protocol name is "8217 Mettler Toledo" (the number comes first). this can of course not be used as a python module or class name, but could be referenced somehow (in a docstring, or maybe even somewhere on the status page?).

Reply:
Added this to the class' docstring. Not sure if sufficient.